src/main/java/pixelperfect/Game.java

```diff
@@ -33,7 +33,7 @@ public static void main(String[] args) {
    */
   @Override
   public void simpleInitApp() {
-    spaceship = new Spaceship();
+		spaceship = Spaceship.getInstance();
```

Comment:
You are right, I think this is not the time yet to look at (possible) Singletons.